utils/install-dependencies.sh

```diff
@@ -24,10 +24,9 @@ function install_dependencies_with_yum() {
     # add OpenResty source
     sudo yum install yum-utils
     sudo yum-config-manager --add-repo "https://openresty.org/package/${1}/openresty.repo"
-    sudo yum check-update
```

Comment:
`sudo yum check-update || true` would be better?

Reply:
IMO, `yum check-update` check whether there is a update or not would't bring any benefit for `install_dependencies_with_yum`.
`yum` don't need to update the repo cache after adding a new repo like `apt` does.
Shall we remove it ?